lib/engine/game/base.rb

```diff
@@ -2760,6 +2760,18 @@ def train_limit_header
       def train_power?
         false
       end
+
+      # This is a bit of a hack to get around how 1840 and 18USA have different and incompatible opinions on what the
+      #  option to use the scrap_trains action in a merger round means
+      def use_1840_style_merger_round_scrap_trains?
+        true
+      end
+
+      # If true, instead of doing a relatively big view with a header and corporation context for scrapping trains, only a
+      # button [{scrap}] will be displayed, where {scrap} is taken from the current step's scrap_info(train) method
+      def use_compact_scrap_trains_view
```

Comment:
why? can we just unify it?

Reply:
Matthew is trying to add a button under the company (aka corporate stock round actions). The default is a whole scrap trains section, much like buy trains. I don't think the two are very compatible, but open to ideas.